osidb/tests/endpoints/flaws/test_flaws.py

```diff
@@ -819,13 +819,11 @@ def test_list_flaws_include_and_exclude_fields(self, auth_client, test_api_uri):
             + [f"affects.trackers.{field}" for field in tracker_include_fields]
         )
 
-        flaw_exclude_fields = ["resolution"]
```

Comment:
This removal gets rid of part of the `exclude_fields` functionality test, can you instead of removing this list entirely supply there a different field name? For example `cve_id`?

Reply:
Sure! just pushed the changes